include/ibm/management_console_rest.hpp

```diff
@@ -879,10 +879,10 @@ void handleCsrRequest(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
                 }
 
                 timeout->cancel();
-                asyncResp->res.result(
-                    boost::beast::http::status::internal_server_error);
-                asyncResp->res.jsonValue["Description"] =
-                    "Timed out waiting for HostInterface to serve request";
+                redfish::messages::serviceTemporarilyUnavailable(
+                    asyncResp->res, std::to_string(60));
+                BMCWEB_LOG_INFO
+                    << "Timed out waiting for HostInterface to serve request";
             };
 
             timeout->async_wait(timeoutHandler);
```

Comment:
Please add an info log at the line 905 - once we get the certificate uploaded successfully. This is to enhance the bmcweb trace for the future maintenance